client/src/js/components/bhSoldOut/bhSoldOut.js

```diff
@@ -0,0 +1,48 @@
+angular.module('bhima.components')
+  .component('bhSoldOut', {
+    templateUrl : 'js/components/bhSoldOut/bhSoldOut.html',
+    controller  : bhSoldOutController,
+    transclude  : true,
+    bindings    : {
+      depotUuid : '<',
+      date : '<',
+    },
+  });
+
+bhSoldOutController.$inject = ['StockService', 'moment', 'NotifyService'];
+
+/**
+ * service or depot selection component
+ */
+function bhSoldOutController(Stock, moment, Notify) {
+  const $ctrl = this;
+  $ctrl.soldOutInventories = [];
+
+  $ctrl.$onInit = () => {
+    stockOut();
+  };
+
+  $ctrl.$onChanges = () => {
+    stockOut();
+  };
+
+  /**
+   * @function stockOut
+   * get stock out inventories for a depot
+   */
+  function stockOut() {
+    const dateTo = $ctrl.date || new Date();
+    if (!$ctrl.depotUuid) return;
```

Comment:
Can you include a loading indicator?  When the depot changes, it would be nice to show a "loading..." indicator.